### PR TITLE
Style vendor, coder insanity edition

### PR DESCRIPTION
**Pull request by Wouju**
## About The Pull Request

Adds a new vendor with VERY stylish clothing

also added newline for voidsuits cause there wasn't one earlier

image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image

image

## Why It's Good For The Game

It's not

Requested by Grey on discord, also option to get stylish clothing in game

## Changelog
:cl: Code - Wouju, Sprites - InfraredBaron
add: Added new stylish clothing in a stylish vendor in a stylish place
imageadd: added new stylish icons for stylish clothes that are contained within a stylish vendor
/:cl:

### CODE REVIEW

**Comment by Omni44** [comment]
pxO7M0e

File: code/modules/clothing/head/misc.dm [hunk 1]
```diff
@@ -260,3 +260,71 @@
...
+	price_tag = 10000
+	style = STYLE_HIGH
+	armor = list(
+		melee = 30,
+		bullet = 25,
+		energy = 25,
+		bomb = 0,
+		bio = 0,
+		rad = 0
+	)
```
Comment (TheShown911) [inline code/modules/clothing/head/misc.dm:330]: No newline.

**Review by TheShown911 — COMMENTED** [review]
Think you messed up the map merger.

File: code/modules/clothing/head/misc.dm (hunk 1 above)
Comment (Wouju) [reply code/modules/clothing/head/misc.dm]: done

**Comment by Wouju** [comment]
reverted the map shit at the moment since it seems i am cursed

File: code/game/machinery/vending.dm [hunk 2]
```diff
@@ -1561,6 +1561,68 @@
 	idle_power_usage = 211
 	vendor_department = DEPARTMENT_OFFSHIP
 
+/obj/machinery/vending/style
+	name = "Style vendor"
```
Comment (TheShown911) [inline code/game/machinery/vending.dm:1565]: Get a better name.

Comment (drexample) [reply code/game/machinery/vending.dm:1565]: Dress-O-Matic

File: code/game/machinery/vending.dm [hunk 3]
```diff
@@ -1561,6 +1561,68 @@
...
+		/obj/item/clothing/under/grey = 600,
+		/obj/item/clothing/under/black = 600,
+		/obj/item/clothing/under/dress = 600,
+		/obj/item/clothing/under/dress/white = 600,
+		/obj/item/clothing/under/helltaker = 600,
+		/obj/item/clothing/under/johnny = 750,
+		/obj/item/clothing/under/raider = 750,
+		/obj/item/clothing/suit/storage/triad = 1200,
+		/obj/item/clothing/suit/storage/akira = 750,
+		/obj/item/clothing/head/skull/drip = 100000,
```
Comment (Firefox13) [inline code/game/machinery/vending.dm:1619]:
```suggestion
		/obj/item/clothing/head/skull/drip = 100000
```

File: code/game/machinery/vending.dm [hunk 4]
```diff
@@ -1561,6 +1561,68 @@
...
+		/obj/item/clothing/under/green = 4,
+		/obj/item/clothing/under/grey = 4,
+		/obj/item/clothing/under/black = 4,
+		/obj/item/clothing/under/dress = 4,
+		/obj/item/clothing/under/dress/white = 4,
+		/obj/item/clothing/under/helltaker = 4,
+		/obj/item/clothing/under/johnny = 3,
+		/obj/item/clothing/under/raider = 3,
+		/obj/item/clothing/suit/storage/triad = 2,
+		/obj/item/clothing/suit/storage/akira = 2,
```
Comment (Firefox13) [inline code/game/machinery/vending.dm:1593]:
```suggestion
		/obj/item/clothing/suit/storage/akira = 2
```

File: code/modules/clothing/masks/miscellaneous.dm [hunk 5]
```diff
@@ -170,6 +170,35 @@
...
+	item_state = "blackscarf"
+	style = STYLE_HIGH
+	price_tag = 100
+
+/obj/item/clothing/mask/scarf/style/bluestyle
+	name = "blue scarf"
+	desc = "A stylish, blue scarf."
+	icon_state = "bluescarf"
+	item_state = "bluescarf"
+	style = STYLE_HIGH
```
Comment (Firefox13) [inline code/modules/clothing/masks/miscellaneous.dm:186]: The style is already defined in its parent.
```suggestion

```

File: code/modules/clothing/masks/miscellaneous.dm [hunk 6]
```diff
@@ -170,6 +170,35 @@
...
+
+/obj/item/clothing/mask/scarf/style/yellowstyle
+	name = "yellow scarf"
+	desc = "A stylish, yellow scarf."
+	icon_state = "yellowscarf"
+	item_state = "yellowscarf"
+	style = STYLE_HIGH
```
Comment (Firefox13) [inline code/modules/clothing/masks/miscellaneous.dm:193]: Ditto
```suggestion

```

File: code/modules/clothing/masks/miscellaneous.dm [hunk 7]
```diff
@@ -170,6 +170,35 @@
...
+
+/obj/item/clothing/mask/scarf/style/redstyle
+	name = "red scarf"
+	desc = "A stylish, red scarf."
+	icon_state = "redscarf"
+	item_state = "redscarf"
+	style = STYLE_HIGH
```
Comment (Firefox13) [inline code/modules/clothing/masks/miscellaneous.dm:200]:
```suggestion

```
Ditto

File: code/modules/clothing/head/misc.dm [hunk 8]
```diff
@@ -260,3 +260,71 @@
...
+
+/obj/item/clothing/head/skull/black
+	name = "black skull"
+	desc = "This is actually a black plastic skull, don't expect much protection."
+	icon_state = "skull-black"
+	item_state = "skull-black"
+	price_tag = 300
+	style = STYLE_HIGH
```
Comment (Firefox13) [inline code/modules/clothing/head/misc.dm:313]:
```suggestion

```
The style is already defined in its parent

File: code/game/machinery/vending.dm (hunk 2 above)
Comment (Wouju) [reply code/game/machinery/vending.dm:1565]: Asters Guild Style-o-matic

anyone who disagrees is a pee pee poo poo head

**Review by Firefox13 — APPROVED** [review]

**Pull request merged**